src/main/java/eu/xenit/gradle/alfresco/DockerAlfrescoPlugin.java

```diff
@@ -138,7 +141,15 @@ private void updateDockerFileTask(Project project1, DockerfileWithWarsTask docke
         for(WarLabelOutputTask task: outputTasks) {
             mergeWarsTask.withLabels(task);
         }
-        mergeWarsTask.setInputWars(() -> outputTasks.stream().map(WarOutputTask::getOutputWar).collect(Collectors.toList()));
+        mergeWarsTask.setInputWars(
+                () -> Stream.concat(Stream.of(baseWar.getSingleFile()),
+                        outputTasks.stream().map(WarOutputTask::getOutputWar)).collect(Collectors.toList()));
+        
+        // Mark merged war as artifact
+        final String artifactConfiguration = "artifact" + warName + "War";
+        project.getConfigurations().create(artifactConfiguration);
+        project.getArtifacts().add(artifactConfiguration, mergeWarsTask.getOutputWar(),
+                configurablePublishArtifact -> configurablePublishArtifact.setName(warName.toLowerCase()));
```

Comment:
The artifactId of the artifact is forced to `alfresco`, but it should be left user configurable.

The extension of the artifact should be configured to be `war`, else it will be published as a `jar` file